Tribler/Core/TorrentChecker/torrent_checker.py

```diff
@@ -341,7 +341,7 @@ def _create_session_for_request(self, infohash, tracker_url):
                 break
 
             #TODO(ardhi) : quickfix for etree.org can't handle multiple infohash in single call
-            if session.can_add_request() and "etree" not in session.tracker_url :
+            if session.can_add_request() and "etree" not in session.tracker_url:
```

Comment:
@lfdversluis, any comment about this? I need the etree tracker for testing but this is just temporary hack :D, so maybe you can generalize this somehow.